### PR TITLE
[DOCS-6397] Added API section

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Comment by rallegre** [comment]
@oliveralfresco2018 the URL you used are not valid. Please check what I wrote in my DOC ticket and re use same ones. Thanks

Application runtime bundle: {domain-name}/{application-name}/rb/swagger-ui/
Application query service: {domain-name}//{application-name}/query/swagger-ui/
GraphQL
The query service can also use GraphQL to expand on querying. It can be accessed at the following URL:
{domain-name}/{application-name}/notifications/graphiql

**Comment by oliveralfresco2018** [comment]
They've been adjusted.
Cheers

**Comment by oliveralfresco2018** [comment]
@rallegre good to go :-)

**Review by rallegre — COMMENTED** [review]
OK for me

File: process-automation/latest/develop/index.md [hunk 1]
```diff
@@ -213,6 +213,16 @@ To include custom form fields within a form, the [form field customizations](htt
...
 
+
+## REST API
+
+The REST APIs are accessed differently depending on whether a service is an application or platform-specific. For application endpoints, the application name will form part of the endpoint.
+
+The OpenAPI specifications for application endpoints require the `{application-name}` element in the URL:
+
+* Application runtime bundle: `{domain-name}/{application-name}/rb/swagger-ui/`.
+* Application query service: `{domain-name}//{application-name}/query/swagger-ui/GraphQLThe`. The query service can also use GraphQL to expand the querying and can be accessed at: `{domain-name}/{application-name}/notifications/graphiql`.
```
Comment (anxumalo) [inline process-automation/latest/develop/index.md:224]: I think there's a stray "The" at the end of the GraphQL endpoint: `../query/swagger-ui/GraphQLThe`?

```suggestion
* Application runtime bundle: `{domain-name}/{application-name}/rb/swagger-ui/`.
* Application query service: `{domain-name}//{application-name}/query/swagger-ui/GraphQL`.

The query service can also use GraphQL to expand the querying and can be accessed at: `{domain-name}/{application-name}/notifications/graphiql`.
```

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**